concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-190	190	(is) dry (feminine)	(est) sèche	sĭcca	64		
+Gauchat-1925-480-191	191	What do you want?	Que veux-tu?	quĭd vŏles tū	64		
+Gauchat-1925-480-192	192	I would like	Je voudrais	volere + habēbam	64		
+Gauchat-1925-480-193	193	some salt	du sel	sal	66	1274	SALT
+Gauchat-1925-480-194	194	some pepper	du poivre	pĭper	66	1385	PEPPER
+Gauchat-1925-480-195	195	one leaf	une feuille	fŏlia	66	628	LEAF
+Gauchat-1925-480-196	196	some cabbage	de chou	caule	66	774	CABBAGE
+Gauchat-1925-480-197	197	a wreath	une gerbe	garba	66	2589	WREATH
+Gauchat-1925-480-198	198	some straw	de paille	palea	66		
+Gauchat-1925-480-199	199	a bag	un sac	saccu	68	2468	BAG
+Gauchat-1925-480-200	200	some oat	d’avoine	avēna	68	60	OAT
+Gauchat-1925-480-201	201	some flour	de farine	farina	68	1594	FLOUR
+Gauchat-1925-480-202	202	an armful	une brassée	braciata	68		
+Gauchat-1925-480-203	203	some armfuls	(des) brassées	braciatas	68		
+Gauchat-1925-480-204	204	of wood	de bois	bŏscu	68	1803	WOOD
+Gauchat-1925-480-205	205	(a) shovel	(une) pelletée	palata	70		
+Gauchat-1925-480-206	206	(some) shovels	(des) pelletées	palatas	70		
+Gauchat-1925-480-207	207	of coal	de charbon	carbōne	70		COAL
+Gauchat-1925-480-208	208	a handful	une poignée	pugnata	70		
+Gauchat-1925-480-209	209	(some) handfuls	(des) poignées	pugnatas	70		
+Gauchat-1925-480-210	210	of ash	de cendres	cĭneres	70	646	ASH
+Gauchat-1925-480-211	211	I had	J’avais	habēbam	72		
+Gauchat-1925-480-212	212	hunger	faim	fame	72	1864	HUNGER
+Gauchat-1925-480-213	213	and thirst	et soif	sĭte	72	848	THIRST
+Gauchat-1925-480-214	214	I have eaten	J’ai mangé	manducatu	72		
+Gauchat-1925-480-215	215	some bread	du pain	pane	72	1368	BREAD
+Gauchat-1925-480-216	216	(some) honey	(du) miel	mĕl	72	942	HONEY
+Gauchat-1925-480-217	217	an egg	un œuf	ŏvu	74	744	EGG
+Gauchat-1925-480-218	218	raw	cru	crūdu	74	1959	RAW
+Gauchat-1925-480-219	219	(some) meat	(de la) viande	carne	74	634	MEAT
+Gauchat-1925-480-220	220	raw (feminine)	crue	crūda	74		
+Gauchat-1925-480-221	221	(some) cheese	(du) fromage	formaticu	74	1569	CHEESE
+Gauchat-1925-480-222	222	(some) "serac"	(du) „sérac“	seraceu	74		
+Gauchat-1925-480-223	223	half	la moitié	medietate	76	816	HALF
+Gauchat-1925-480-224	224	of an apple	d’une pomme	pōma	76	1320	APPLE
+Gauchat-1925-480-225	225	cooked	cuite	cŏcta	76	269	COOKED
+Gauchat-1925-480-226	226	(the apple is) eaten	(la pomme est) mangée	manducata	76		
+Gauchat-1925-480-227	227	The two halves	Les deux moitiés	dŭas medietates	76		
+Gauchat-1925-480-228	228	(are) eaten	(sont) mangées	manducatas	76		
+Gauchat-1925-480-229	229	I needed	Il m’a fallu	fallūtu	78		
+Gauchat-1925-480-230	230	to drink	boire	bĭbere	78	1401	DRINK
+Gauchat-1925-480-231	231	some milk	du lait	lacticĕllu	78	635	MILK
+Gauchat-1925-480-232	232	some whey	du petit-lait	lactata	78		
+Gauchat-1925-480-233	233	The wine	Le vin	vīnu	78	1524	WINE
+Gauchat-1925-480-234	234	is clear	est clair	claru	78	250	LIGHT (COLOR)
+Gauchat-1925-480-235	235	lukewarm	tiède	tĕpidu	80	1232	WARM
```

Comment:
lukewarm is not warm.

Reply:
Yes, true. Tiède means lukewarm, so this should be unmapped.